.github/workflows/release_cut_new.yml

```diff
@@ -140,10 +143,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # actions/checkout elides tags, fetch them primarily for releases
```

Comment:
I think this is going to break the executable's version information, since it uses `git tag` internally.